### PR TITLE
docs(react-context-selector): fixed code example issue

### PR DESCRIPTION
**Pull request by mathis-m**
This is a minor change to fix the example listed in the readme of react-context-selector

### CODE REVIEW

**Review by layershifter — APPROVED** [review]

**Comment by mathis-m** [comment]
@microsoft-github-policy-service agree

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">79</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">75</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/Avatar-mount_base.html">631</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/Avatar-mount.html">632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/Button-mount_base.html">303</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/Button-mount.html">313</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/Field-mount_base.html">1160</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/Field-mount.html">1165</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProvider-mount_base.html">739</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProvider-mount.html">729</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProviderWithTheme-mount_base.html">86</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProviderWithTheme-mount.html">84</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">34</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">39</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">79</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">75</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/MakeStyles-mount_base.html">903</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/MakeStyles-mount.html">892</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/Persona-mount_base.html">1797</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/Persona-mount.html">1754</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/SpinButton-mount_base.html">1430</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/SpinButton-mount.html">1380</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/SwatchPicker-mount_base.html">1685</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32207/perf-test-react-components/SwatchPicker-mount.html">1696</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-react-context-selector-d5e34afc-03aa-408c-8192-bb66d5b01f19.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-context-selector-d5e34afc-03aa-408c-8192-bb66d5b01f19.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32207&commitId=f2a38a330e7b2465ab92669d265a532e21c835e3&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|4| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Pull request merged**